Docs/Configuration.tex

```diff
@@ -1459,6 +1459,22 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   slide values are usable!} message in the debug log. If the message is present,
   this option is to be enabled.
 
+\item
+  \texttt{ProvideMaxSlide}\\
+  \textbf{Type}: \texttt{plist\ integer}\\
+  \textbf{Failsafe}: \texttt{0}\\
+  \textbf{Description}: Provide maximum KASLR slide when higher ones are unavailable.
+
+  This option overrides the maximum slide instead of \texttt{TOTAL\_SLIDE\_NUM} (256)
+  when \texttt{ProvideCustomSlide} is enabled. When generating random slides, this
+  option can avoid bad slides that are unavailable and cause boot failure.
```

Comment:
Could you add the following afterwards? It is believed that modern firmwares allocate pool memory from top to bottom, effectively resulting in free memory at the time of slide scanning being later used as temporary memory during kernel loading.